src/main/scala/is/hail/io/vcf/LoadGDB.scala

```diff
@@ -0,0 +1,175 @@
+package is.hail.io.vcf
+
+import com.intel.genomicsdb.GenomicsDBFeatureReader
+import htsjdk.variant.vcf.VCFHeader
+import is.hail.HailContext
+import is.hail.annotations._
+import is.hail.expr.{TStruct, _}
+import is.hail.utils._
+import is.hail.variant.{Locus, VSMLocalValue, VSMMetadata, Variant, VariantSampleMatrix}
+import org.apache.spark.storage.StorageLevel
+import org.json4s._
+
+import scala.collection.JavaConversions._
+import scala.collection.JavaConverters.asScalaIteratorConverter
+import java.io.{File, FileWriter}
+
+case class QueryJSON(workspace: String,
+                     array: String,
+                     vid_mapping_file: String,
+                     callset_mapping_file: String,
+                     vcf_header_filename: Option[String],
+                     //type List[List[Any]] b/c genomicsDB accepts both single columns and ranges of columns to query
+                     query_column_ranges: List[List[Any]] = List(List(List(0, 17421565))), //TODO: make this a parameter?
+                     reference_genome: String = "",
+                     query_attributes: List[String] = List(),
+                     query_row_ranges: Option[List[List[Int]]] = None,
+                     max_diploid_alt_alleles_that_can_be_genotyped: Option[Int] = None,
+                     vcf_output_filename: Option[List[String]] = None,
+                     vcf_output_format: Option[String] = None,
+                     produce_GT_field: Boolean = true,
+                     index_output_VCF: Option[Boolean] = None,
+                     combined_vcf_records_buffer_size_limit: Option[Int] = None)
+
+object LoadGDB {
+
+  def createQueryJSON(tiledbworkspace: String,
+                      arrayName: String,
+                      vid_mapping_file: String,
+                      callsets_mapping_file: String,
+                      vcfHeaderPath: Option[String]): File = {
+    val tempFile = File.createTempFile("sample2query", ".json")
+    jackson.Serialization.writePretty(QueryJSON(tiledbworkspace,
```

Comment:
let's use write instead of writePretty since a human need not read the query json